tests/k8st/tests/test_bgp_advert_v6.py

```diff
@@ -161,7 +161,7 @@ def get_svc_cluster_ip(self, svc, ns):
                        (svc, ns)).strip()
 
     def assert_ecmp_routes(self, dst, via):
-        matchStr = dst + " proto bird metric 1024 "
+        matchStr = dst + " proto bird metric 1024 pref medium"
```

Comment:
I'm not sure why this has suddenly appeared in `ip -6 r` output. It does seem alright - this specifies the PREF string specifies the route priority and medium is "default priority".